src/user/header/header-titles.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Grid, Paper } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyle } from 'user/header/style'
+
+export const HeaderTitles = ({ items }) => <Grid item xs={3}>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125785818-e172f7c4-cd4a-4642-aa45-4b263733abbe.png)
Since this component represents a single header, let's singularize it .

Reply:
Done.